source/codegen/metadata/nifgen/functions.py

```diff
@@ -1058,6 +1058,72 @@
         ],
         'returns':'ViStatus'
     },
+    'CreateAdvancedArbSequence':{
+        'codegen_method': 'CustomCode',
+        'parameters':[
+            {
+                'name':'vi',
+                'direction':'in',
+                'type':'ViSession'
+            },
+            {
+                'name':'sequenceLength',
+                'direction':'in',
+                'type':'ViInt32'
+            },
+            {
+                'name':'waveformHandlesArray',
+                'direction':'in',
+                'type':'ViInt32[]',
+                'size':{
+                    'mechanism':'len',
+                    'value':'sequenceLength'
+                }
+            },
+            {
+                'name':'loopCountsArray',
+                'direction':'in',
+                'type':'ViInt32[]',
+                'size':{
+                    'mechanism':'len',
+                    'value':'sequenceLength'
+                }
+            },
+            {
+                'name':'sampleCountsArray',
+                'direction':'in',
+                'type':'ViInt32[]',
+                'size':{
+                    'mechanism':'len',
+                    'value':'sequenceLength'
+                }
+            },
+            {
+                'name':'markerLocationArray',
+                'direction':'in',
+                'type':'ViInt32[]',
+                'size':{
+                    'mechanism':'len',
+                    'value':'sequenceLength'
+                }
+            },
+            {
+                'name':'coercedMarkersArray',
+                'direction':'out',
+                'type':'ViInt32[]',
+                'size':{
+                    'mechanism':'custom-code',
+                    'value':'sequenceLength'
```

Comment:
@doshirohan , @ganeshnie , in [nimi-python](https://github.com/ni/nimi-python/blob/master/src/nifgen/metadata/functions.py#L584), the value has the following python snippet:
`'value': '(0 if marker_location_array is None else len(marker_location_array))'`.
For gRPC, is it okay to mention as sequenceLength? If not, please suggest what can be used.